integration/env_test.go

```diff
@@ -0,0 +1,183 @@
+package integration_test
```

Comment:
What do you think about actually making this utility public in Starport repo so we can use it here by importing? -Tho this can be any time, I think for now it is OK to duplicate the code here.

Reply:
That's a good idea. Let's see in the future if we will need this again :)